app/src/main/java/com/hedvig/app/feature/keygear/KeyGearValuationActivity.kt

```diff
@@ -151,6 +174,42 @@ class KeyGearValuationActivity : BaseActivity(R.layout.activity_key_gear_valuati
         }
     }
 
+    private fun animateDateDown() {
+        dateInput.spring(
+            SpringAnimation.TRANSLATION_Y,
+            SpringForce.STIFFNESS_HIGH,
+            SpringForce.DAMPING_RATIO_NO_BOUNCY
+        ).animateToFinalPosition(getNoCoverageHeight())
+
+        saveContainer.spring(
+            SpringAnimation.TRANSLATION_Y,
+            SpringForce.STIFFNESS_HIGH,
+            SpringForce.DAMPING_RATIO_NO_BOUNCY
+        ).animateToFinalPosition(getNoCoverageHeight())
+    }
+
+    private fun animateDateUp() {
+        dateInput.spring(
+            SpringAnimation.TRANSLATION_Y,
+            SpringForce.STIFFNESS_HIGH,
+            SpringForce.DAMPING_RATIO_NO_BOUNCY
+        ).animateToFinalPosition(-(getNoCoverageHeight() / 50f))
+
+        saveContainer.spring(
+            SpringAnimation.TRANSLATION_Y,
+            SpringForce.STIFFNESS_HIGH,
+            SpringForce.DAMPING_RATIO_NO_BOUNCY
+        ).animateToFinalPosition(-(getNoCoverageHeight() / 50f))
+    }
+
+    private fun getNoCoverageHeight(): Float {
+        noCoverage.measure(
+            LinearLayout.LayoutParams.MATCH_PARENT,
```

Comment:
I'm not sure that this is correct, measure takes `MeasureSpec`s, which dont map this well to layout params afaik